manifests/integreatly-monitoring/1.6.1/application-monitoring-operator.v1.6.1.clusterserviceversion.yaml

```
@@ -0,0 +1,354 @@
+                ports:
+                - containerPort: 8383
+                  name: metrics
+                resources: {}
+              serviceAccountName: application-monitoring-operator
+      permissions:
+      - rules:
+        - apiGroups:
+          - ""
+          resources:
+          - pods
+          - services
+          - services/finalizers
+          - endpoints
+          - persistentvolumeclaims
+          - events
+          - configmaps
+          - secrets
+          - serviceaccounts
+          verbs:
+          - '*'
+        - apiGroups:
+          - apps
+          resources:
+          - deployments
+          - deployments/finalizers
+          - daemonsets
+          - replicasets
+          - statefulsets
+          verbs:
+          - '*'
+        - apiGroups:
+          - monitoring.coreos.com
+          resources:
+          - alertmanagers
+          - prometheuses
+          - prometheusrules
+          - servicemonitors
+          verbs:
+          - '*'
+        - apiGroups:
+          - applicationmonitoring.integreatly.org
+          resources:
+          - applicationmonitorings
+          - applicationmonitorings/finalizers
+          - blackboxtargets
+          - blackboxtargets/finalizers
+          verbs:
+          - '*'
+        - apiGroups:
+          - integreatly.org
+          resources:
+          - grafanadatasources
+          - grafanadatasources/status
+          - grafanadashboards
+          - grafanadashboards/status
+          - grafanas
+          - grafanas/status
+          - grafanas/finalizers
+          - grafanadatasources/finalizers
+          - grafanadashboards/finalizers
+          verbs:
+          - '*'
+        - apiGroups:
+          - route.openshift.io
+          resources:
+          - routes
+          - routes/custom-host
+          verbs:
+          - '*'
+        - apiGroups:
+          - rbac.authorization.k8s.io
+          resources:
+          - rolebindings
+          - roles
+          verbs:
+          - '*'
+        - apiGroups:
+          - extensions
+          resources:
+          - ingresses
+          verbs:
+          - '*'
+        - apiGroups:
+          - authentication.k8s.io
+          resources:
+          - tokenreviews
+          verbs:
+          - create
+        - apiGroups:
+          - authorization.k8s.io
+          resources:
+          - subjectaccessreviews
+          verbs:
+          - create
+        serviceAccountName: application-monitoring-operator
+    strategy: deployment
+  installModes:
+  - supported: true
+    type: OwnNamespace
+  - supported: true
+    type: SingleNamespace
+  - supported: false
+    type: MultiNamespace
+  - supported: true
+    type: AllNamespaces
+  keywords:
+  - Prometheus
+  - Grafana
+  - Monitoring
+  - AlertManager
+  maintainers:
+  - email: placeholder@email.com
+    name: Integreatly
+  maturity: alpha
+  provider:
+    name: Red Hat
+  replaces: application-monitoring-operator.v1.5.1
```

```suggestion
  replaces: application-monitoring-operator.v1.6.0
```
Should this be replaces `1.6.0` ? :thinking:

---

Yep you're right.